bot-components/Utils.mli

```diff
@@ -0,0 +1,28 @@
+val f : ('a, unit, string) format -> 'a
+
+val string_match : regexp:string -> string -> bool
+
+val headers :
+  bot_info:Bot_info.bot_info -> (string * string) list -> Cohttp.Header.t
+
+val print_response : Cohttp.Response.t * Cohttp_lwt.Body.t -> unit Lwt.t
+
+val send_request :
+     bot_info:Bot_info.bot_info
+  -> body:Cohttp_lwt.Body.t
+  -> uri:Uri.t
+  -> (string * string) list
+  -> unit Lwt.t
+
+val get_backport_info :
```

Comment:
This function is only used in `GitHub_queries` as far as I can see. So instead of moving it to `Utils`, you could simply have not exported it (and also possibly renamed it so that its name doesn't suggest that it is one of the functions that is exported).